components/html-block/html-block.js

```diff
@@ -208,6 +208,7 @@ class HtmlBlock extends LitElement {
 
 		this._renderersProcessedResolve = undefined;
 		this._renderersProcessedPromise = new Promise(resolve => this._renderersProcessedResolve = resolve);
+		this._renderContainerRef = createRef();
```

Comment:
This isn't strictly needed, but I do think it's a more stable way to fetch this reference.